package.json

```diff
@@ -4,7 +4,8 @@
     "start": "node app.js",
     "dev": "NODE_ENV=development hof-build watch",
     "test": "NODE_ENV=test npm-run-all test:unit lint",
-    "test:unit": "nyc _mocha",
+    "test:unit": "_mocha",
+    "test:coverage": "nyc _mocha",
```

Comment:
probably remove `_mocha` from the coverage test or you could have a separate script to do both